common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -309,7 +309,8 @@
                 error = function() {
                     self.renderThreads();
                     DiscussionUtil.discussionAlert(
-                        gettext('Sorry'), gettext('We had some trouble loading more posts. Please try again.')
+                        gettext('Error'),
+                        gettext('New posts could not be loaded. Refresh the page to try again.')
```

Comment:
@andy-armstrong Is this message really relating to "new" posts, or should we say "Additional posts could not be loaded."